torch_xla/csrc/tensor_methods.cpp

```diff
@@ -1368,6 +1369,10 @@ void XLATensor::log1p_(XLATensor& input) {
   input.SetIrValue(ir::ops::Log1p(input.GetIrValue()));
 }
 
+XLATensor XLATensor::logdet(const XLATensor& input) {
+  return input.CreateFrom(ir::ops::LogDet(input.GetIrValue()));
```

Comment:
Does this even build?
That op takes no metadata, so it should be in ops.cpp/h ... so the call here is correct.
But you created a separate cpp/h files, which is not needed, and eventually for that case here you should have called MakeNode<>().

Reply:
Oh I'm sorry, I missed that. Please allow me to fix it. Thank you.